M2/Macaulay2/d/actors4.d

```diff
@@ -1399,39 +1399,6 @@ subsetRRi(e:Expr):Expr := (
          else WrongArg(1,"a pair of integral, rational, real numbers or intervals"));
```

Comment:
An "interval" is primarily a real number, so such things are implicitly included in the phrase "real numbers".